inclusive-design/wecount.inclusivedesign.ca
# feat: replace sass build

## Description

This PR replaces eleventy-plugin-sass with the Sass compilation technique used in https://github.com/maxboeck/eleventastic.

## Steps to test

1. `npm install`
2. `npm start`
3. `npm build`

**Expected behavior:** Sass is compiled in dev mode and changes trigger a live reload. Sass builds and is minified in production mode. 

## Additional information

Not applicable.

## Related issues

Not applicable.
